md/dates-management-tutorial.md

```diff
@@ -0,0 +1,221 @@
+And since the contract input is of type **DATE**, the ISO 8601 formatted string will be automatically converted into a `java.util.Date` by the task execution REST API. 
+:::
+
+#### Retrieve the date from the database and display it
+
+In another step form (or in any other page) you may want to retrieve the date and display it.  
+In order to do so:  
+1. Create a new task named *Review birth date* after the existing one
+1. In the task **Execution** tab, **Form** section, generate the form by clicking on the **pencil** icon
+1. In the UI Designer, define a variable to retrieve the *Employee* currently stored using the context.  
+In order to do so, proceed the same way as you did for the previous step form:  
+   1. In the **Variables** panel at the bottom of the editor, click on **Create a new variable**
+   1. Enter the name *employee* for the new variable
+   1. select the **type** *external API*
+   1. Set the **API URL** of the variable to `../{{context.employee_ref.link}}`
+1. From the **Form editor** palette, drop a **Date picker** widget on the whiteboard
+1. Set the property **Read-only** to *yes*
+1. Set its **Value** to `employee.birthdate`
+
+You may prefer to display it in some text. In order to do so:  
+1. Drop a **Text** widget on the whiteboard
+1. Replace the default **Text** property value with an expression using a *[date](https://docs.angularjs.org/api/ng/filter/date)* filter to format the date the way you want. E.g., `{{employee.birthdate | date:'MM/dd/yyyy':'UTC'}}`  
+
+![Date filter](images/dates-management-tutorial/tuto-dates-date-filter.png) <!--{.img-responsive .img-thumbnail}-->
+
+You can now run your process and see that the date retrieved from the business data database is consistent with the date you chose in the previous step.
+
+## Business object attributes with type STRING
+
+#### Define the business data
+
+1. In the studio, open the Business Data Model editor (in menu **Development** > **Business Data Model** > **Manage...**)
+1. Create a **business object** *Employee* with an attribute *birthdate* of type **STRING**  
+![BDM](images/dates-management-tutorial/tuto-dates-bdm.png) <!--{.img-responsive .img-thumbnail}-->  
+1. Create a new process
+1. In the process **Data** tab, create a new **business variable** *employee* of the type *Employee*
+
+#### Define the contract at pool level, as well as the script to initialize the business variable
+
+1. In the process **Execution** tab > **Contract** section, define the contract inputs using the **Add from data...** button
+1. Select the business variable *employee*
+1. Follow the steps to generate a contract with a COMPLEX input *employeeInput* containing a *birthdate* attribute of type **TEXT** (keep the option to generate the variable initialization script automatically)
+
+![Contract](images/dates-management-tutorial/tuto-dates-contract-text.png) <!--{.img-responsive .img-thumbnail}-->
```

Comment:
the contract shows more attributes than expected from the business data chapter.
May be worth adjusting